templates/taxi/driver_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update license number of,Create new driver" }} {{ driver.username }}</h1>
```

Comment:
The `yesno` filter is used to determine the heading based on the presence of `object`. Ensure that `object` is correctly passed to the template context. If `object` is not available, the heading logic might not work as expected. Consider verifying that the `object` context variable is correctly set in the view rendering this template.